src/components/AddPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component, Fragment } from 'react';
+import styled from 'styled-components'
+import { connect } from 'react-redux';
+import FormInfo from './shared/FormInfo';
+
+const StyledSection = styled.section`
+    width: 400px;
+    padding: 25px;
+    margin: 50px auto;
+    border: 3px solid black;
+    background-color: #424242;
+    div {
+        margin-bottom: 10px;
+    }
+`;
+
+class AddPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            namePage: 'Add movie',
+            nameButton: 'Submit',
+        };
+    }
+
+    render() {
+        return (
+            <Fragment>
```

Comment:
Why do u always wrap your components into `<Fragment>` ?
It is only necessary when you have to return multiple elements
https://reactjs.org/docs/fragments.html